### PR TITLE
Ran autopep8

### PR DESCRIPTION
**Pull request by externl**
I ran autopep8 over the all the python files. It only performs non-breaking changes. In a follow-up PR I plan to do more of the manual lints.

### CODE REVIEW

**Comment by ReeceHumphreys** [comment]
I think we should be formatting and linting with Ruff https://docs.astral.sh/ruff/. It seems like it has taken off over the past few years and is generally the recommended tool for Python formatting and linting. It is basically a drop in replacement for the Flake8 + Black + isort + ... python linting and formatting toolchain.

I have been testing it locally for the past few days on the metrics project and it is considerably faster than those other tools too!

File: cpp/test/Glacier2/staticFiltering/test.py [hunk 1]
```diff
@@ -298,4 +298,6 @@ def setup(self, current):
         for testcase in testcases:
             self.addTestCase(Glacier2StaticFilteringTestCase(testcase, hostname))
 
-Glacier2StaticFilteringTestSuite(__name__, testcases=[], runOnMainThread=True, options={ "ipv6" : [False] }, multihost=False)
+
+Glacier2StaticFilteringTestSuite(__name__, testcases=[], runOnMainThread=True,
+                                 options={"ipv6": [False]}, multihost=False)
```
Comment (pepone) [inline cpp/test/Glacier2/staticFiltering/test.py:303]: Is this configurable? I prefer one parameter per line when we have to split a long line.

File: cpp/test/IceGrid/update/test.py [hunk 2]
```diff
@@ -10,13 +10,16 @@ def setupClientSide(self, current):
...
+    "IceGridNodeExe": IceGridNode().getCommandLine(current),
+    "ServerDir": current.getBuildDir("server"),
+    "TestDir": "{testdir}"
 }
 
+
 if isinstance(platform, Windows) or os.getuid() != 0:
-    TestSuite(__file__, [IceGridUpdateTestCase(application=None, client=IceGridClient(props=clientProps))], multihost=False)
+    TestSuite(__file__, [IceGridUpdateTestCase(application=None,
+              client=IceGridClient(props=clientProps))], multihost=False)
```
Comment (pepone) [inline cpp/test/IceGrid/update/test.py:25]: I don't like this formatting very much

```
TestSuite(__file__, 
          [IceGridUpdateTestCase(application=None, client=IceGridClient(props=clientProps))],
          multihost=False)
```
seems more natural

**Review by pepone — APPROVED** [review]

File: cpp/test/Glacier2/staticFiltering/test.py (hunk 1 above)
Comment (externl) [reply cpp/test/Glacier2/staticFiltering/test.py:303]: For now I think we should just to the defaults. Once all the linting is done we can go back and tweak the style if we really want to.

File: cpp/test/IceGrid/update/test.py (hunk 2 above)
Comment (externl) [reply cpp/test/IceGrid/update/test.py:25]: For now I think we should just to the defaults. Once all the linting is done we can go back and tweak the style if we really want to.

**Pull request merged**